src/constants.ts

```diff
@@ -8,14 +8,24 @@ export enum COMMANDS {
   PARSER = 'PARSER',
   CUSTOM = 'CUSTOM',
 }
-export type FluentBitSchemaType = {
+
+export type FluentBitSection = {
   id: string;
   name?: string;
   command: COMMANDS;
-  optional?: Record<string, unknown>;
-} & { [key: string]: unknown };
+  optional?: { [key: string]: unknown };
+};
+
+export interface FluentBitSchemaType extends FluentBitSection {
+  __filePath: string;
+}
 export const FLUENTBIT_REGEX = /(?<![#][ ]*)\[[A-Z]{1,}\]/g;
 
+/** It will match @includes files as a valid Fluent Bit configuration.
+ *  [Follow this link for an example](https://regex101.com/r/zrSRR2/1)
```

Comment:
Good example.

Maybe add a comment explaining that `@include` cannot be indented otherwise it fails as it is then considered a property of the section you are in.

Reply:
This will work currently. we are capturing the `@includes` in the line.